project/components/resource_request/schemas.py

```diff
@@ -32,6 +32,8 @@ class ResourceRequestSchema(BaseSchema):
     username: str
     requested_for: str
     completed_at: datetime = None
+    message: str = None
```

Comment:
Use optional str typing for nullable values:
```suggestion
    message: str | None = None
```

Reply:
I had to use ugly Unions as this project is on 3.9